apps/portals/e2e/view_github_report.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# View Playwright report stored in GitHub
+#   ./view_github_report.sh [github-repo-owner] [github-run-id]
+#  
+
+# Abort script on errors and unbound variables
+# https://bertvv.github.io/cheat-sheets/Bash.html
+set -o errexit   # abort on nonzero exitstatus
+set -o nounset   # abort on unbound variable
+set -o pipefail  # don't hide errors within pipes
+
+# Set variables
+SCRIPT_PATH="$(cd "$(dirname "${0}")"; echo $(pwd))"
+PROJECT_PATH="$(dirname ${SCRIPT_PATH})"
+REPO_NAME="$(basename "$(dirname "$(dirname "$PROJECT_PATH")")")" # handle monorepo structure
```

Comment:
Fairly similar to the SWC script, with small tweaks to handle the monorepo structure. For example, the repo name used to pull the artifact from GitHub should be the monorepo name (`synapse-web-monorepo`), not the name of the subproject that contains this script (`portals`)